hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -383,6 +383,7 @@
                 export PROJECT="k8s-jkns-e2e-gke-prod"
                 export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/rc"
                 export JENKINS_USE_SERVER_VERSION="y"
+                export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://container.googleapis.com/"
```

Comment:
yeah, the default used to be prod. I guess as long as we've audited all places we expected this to be prod, we're probably OK.